src/tests/functional/plugin/conformance/subgraphs_dumper/include/cache/meta/meta_info.hpp

```diff
@@ -28,7 +28,7 @@ class MetaInfo {
     std::map<std::string, ModelInfo> get_model_info() const;
     std::string get_any_extractor() const { return *extractors.begin(); }
 
-    static MetaInfo read_meta_from_file(const std::string& meta_path);
+    static MetaInfo read_meta_from_file(const std::string& meta_path, size_t model_priority = 1);
```

Comment:
The best option is to add model_priority to meta directly

Reply:
what do you mean ? I need to add method to update model_priority here ?